definitions/variable/tag_electricity-source.yaml

```
@@ -0,0 +1,23 @@
+- Electricity Source:
```

To avoid conflict with my previous comment, I suggest to rename
file name -> `tag_secondary_electricity_sources_additional.yaml`
```suggestion
- Electricity Source By Technology:
```